source/community/reactnative/src/components/actions/action-button.js

```diff
@@ -114,7 +114,8 @@ export class ActionButton extends React.Component {
 			else
 				mergedObject["actionData"] = this.data;
 		}
-		let actionObject = { "type": this.payload.type, "data": mergedObject };
+		const title = this.payload.title
```

Comment:
Shall we have :
`const { title = "",  ignoreInputValidation } = this.payload;`
This also avoid the null check function call on `title`

Reply:
Updated